### ISSUE TITLE
Handle no free disk space error from prd

### ISSUE THREAD

**Issue by gajop** [gajop/spring-launcher#34]
[Sun Mar 15 2020 16:18:06 GMT-0400 (Eastern Daylight Time) warn] [Error] ../../tools/pr-downloader/src/pr-downloader.cpp:262:DownloadStart():Insuffcient free disk space (268 MiB) on C:\Program Files\Beyond-All-Reason\data: 1024 MiB needed
[Error] ../../tools/pr-downloader/src/main.cpp:221:main():Error occurred while downloading: 5

**Comment by abma** [comment]
why is the error message incorrect?

**Comment by abma** [comment]
there is to few disk space...

**Comment by gajop** [comment]
@abma You're right. I think this is trully too few disk space. I've split the "incorrect message" ones to https://github.com/gajop/spring-launcher/issues/57
